client/src/elm/Pages/GlobalCaseManagement/View.elm

```diff
@@ -80,56 +92,55 @@ view language currentDate ( healthCenterId, maybeVillageId ) isChw model db =
-                    viewNutritionPane language currentDate nutritionFollowUps db model
+        nutritionFollowUps =
+            generateNutritionFollowUps currentDate followUpsForResidents
+                |> fillPersonName identity db
 
-                acuteIllnessFollowUps =
-                    generateAcuteIllnessFollowUps currentDate db followUps
-                        |> filterVillageResidents villageId Tuple.second db
+        nutritionFollowUpsPane =
+            viewNutritionPane language currentDate nutritionFollowUps db model
 
-                acuteIllnessFollowUpsPane =
-                    viewAcuteIllnessPane language currentDate acuteIllnessFollowUps db model
+        acuteIllnessFollowUps =
+            generateAcuteIllnessFollowUps currentDate db followUpsForResidents
+                |> fillPersonName Tuple.second db
 
-                prenatalFollowUps =
-                    generatePrenatalFollowUps currentDate db followUps
-                        |> filterVillageResidents villageId Tuple.second db
+        acuteIllnessFollowUpsPane =
+            viewAcuteIllnessPane language currentDate acuteIllnessFollowUps db model
 
-                prenatalFollowUpsPane =
-                    viewPrenatalPane language currentDate prenatalFollowUps db model
+        prenatalFollowUps =
+            generatePrenatalFollowUps currentDate db followUpsForResidents
+                |> fillPersonName Tuple.second db
 
-                panes =
-                    [ ( FilterAcuteIllness, acuteIllnessFollowUpsPane )
-                    , ( FilterAntenatal, prenatalFollowUpsPane )
-                    , ( FilterNutrition, nutritionFollowUpsPane )
-                    ]
-                        |> List.filterMap
-                            (\( type_, pane ) ->
-                                if isNothing model.filter || model.filter == Just type_ then
-                                    Just pane
-
-                                else
-                                    Nothing
-                            )
-            in
-            div [ class "ui unstackable items" ] <|
-                viewFilters language chwFilters model
-                    :: panes
-        )
-        maybeVillageId
-        |> Maybe.withDefault (Pages.PageNotFound.View.viewPage language (SetActivePage PinCodePage) (UserPage GlobalCaseManagementPage))
+        prenatalFollowUpsPane =
+            viewPrenatalPane language currentDate prenatalFollowUps db model
+
+        panes =
+            [ ( FilterAcuteIllness, acuteIllnessFollowUpsPane )
+            , ( FilterAntenatal, prenatalFollowUpsPane )
+            , ( FilterNutrition, nutritionFollowUpsPane )
+            ]
+                |> List.filterMap
+                    (\( type_, pane ) ->
+                        if isNothing model.filter || model.filter == Just type_ then
+                            Just pane
+
+                        else
+                            Nothing
```

Comment:
Is it an error handling case? If it is, we may need a message. If not, ignore my comment.